hashes/src/sha256t.rs

```diff
@@ -228,12 +228,6 @@ macro_rules! sha256t_tag {
 /// BIP-341. Alternatively, you can supply `hash_bytes` to hash raw bytes. If you have the midstate
 /// already pre-computed and prefer **compiler** performance to readability you may use
 /// `raw(MIDSTATE_BYTES, HASHED_BYTES_LENGTH)` instead.
-///
-/// Both visibility modifiers and attributes are optional and passed to inner structs (excluding
-/// `#[hash_newtype(...)]`). The attributes suffer same compiler performance limitations as in
-/// [`hash_newtype`] macro.
```

Comment:
I'm pretty sure I've already said that the perf limitation is still in **this** macro. Is this broken rebase?

Reply:
Ah damn, it was risky rebasing this late yesterday evening, sorry for stealing your time.